packages/storybook/src/community/accordion.stories.tsx

```diff
@@ -0,0 +1,268 @@
+import {
+  AccordionProvider,
+  AccordionSectionProps,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHeader,
+  TableHeaderCell,
+  TableRow,
+} from '@rijkshuisstijl-community/components-react';
+import { Meta, StoryObj } from '@storybook/react';
+import { UtrechtIconChevronDown } from '@utrecht/web-component-library-react';
+import { ReactNode } from 'react';
+interface AccordionStoryProps {
+  label: string;
+  body: ReactNode;
+  expanded?: boolean;
+  expandedAccordion?: boolean;
+  icon?: ReactNode;
+  appearance?: string;
+  sections?: AccordionSectionProps[];
+}
+
+const AccordionStory = ({ expanded, label, body, appearance, icon, sections }: AccordionStoryProps) => (
+  <AccordionProvider icon={icon} appearance={appearance} sections={sections || [{ expanded, label, body }]} />
+);
+
+const meta = {
+  title: 'React Component/Accordion',
```

Comment:
<img width="290" alt="Screenshot 2024-08-02 at 09 24 04" src="https://github.com/user-attachments/assets/717c89b5-3903-4488-adbf-96b5b156f81f">

```suggestion
  title: 'Rijkshuisstijl/Accordion',
```

Reply:
Done :)